modules/openapi-generator/src/main/resources/typescript-fetch/modelGeneric.mustache

```diff
@@ -118,7 +118,7 @@ export function {{classname}}ToJSON(value?: {{classname}} | null): any {
         {{#vars}}
         {{^isReadOnly}}
         {{#isPrimitiveType}}
-        '{{baseName}}': {{#isDateType}}{{^required}}value.{{name}} === undefined ? undefined : {{/required}}({{#isNullable}}value.{{name}} === null ? null : {{/isNullable}}value.{{name}}.toISOString().substr(0,10)){{/isDateType}}{{#isDateTimeType}}{{^required}}value.{{name}} === undefined ? undefined : {{/required}}({{#isNullable}}value.{{name}} === null ? null : {{/isNullable}}value.{{name}}.toISOString()){{/isDateTimeType}}{{^isDateType}}{{^isDateTimeType}}value.{{name}}{{/isDateTimeType}}{{/isDateType}},
+        '{{baseName}}': {{#isDateType}}{{^required}}value.{{name}} === undefined ? undefined : {{/required}}({{#isNullable}}value.{{name}} === null ? null : {{/isNullable}}value.{{name}}.toISOString().substr(0,10)){{/isDateType}}{{#isDateTimeType}}{{^required}}value.{{name}} === undefined ? undefined : {{/required}}({{#isNullable}}value.{{name}} === null ? null : {{/isNullable}}value.{{name}}.toISOString()){{/isDateTimeType}}{{#isArray}}{{#uniqueItems}}{{^required}}value.{{name}} === undefined ? undefined : {{/required}}{{#isNullable}}value.{{name}} === null ? null : {{/isNullable}}Array.from(value.{{name}} as Set<any>){{/uniqueItems}}{{^uniqueItems}}value.{{name}}{{/uniqueItems}}{{/isArray}}{{^isDateType}}{{^isDateTimeType}}{{^isArray}}value.{{name}}{{/isArray}}{{/isDateTimeType}}{{/isDateType}},
```

Comment:
Wouldnt it be simpler to just always run

  Array.from(values)

?

Moreover, i think this statement can be placed on multiple lines. if each line contains only a mustache expression, no additional linebreak is inserted

Reply:
> Wouldnt it be simpler to just always run
> Array.from(values)

I think it would be better to only introduce additional complexity to the generated code if it is needed.

> Moreover, i think this statement can be placed on multiple lines. if each line contains only a mustache expression, no additional linebreak is inserted

I tried to do that, but without much success. I tried to do it like this:
```mustache
{{#isPrimitiveType}}
{{#isDateType}}
'{{baseName}}': {{^required}}value.{{name}} === undefined ? undefined : {{/required}}({{#isNullable}}value.{{name}} === null ? null : {{/isNullable}}value.{{name}}.toISOString().substr(0,10)),
{{/isDateType}}
{{#isDateTimeType}}
'{{baseName}}': {{^required}}value.{{name}} === undefined ? undefined : {{/required}}({{#isNullable}}value.{{name}} === null ? null : {{/isNullable}}value.{{name}}.toISOString()),
{{/isDateTimeType}}
{{#isArray}}
'{{baseName}}': {{#uniqueItems}}{{^required}}value.{{name}} === undefined ? undefined : {{/required}}{{#isNullable}}value.{{name}} === null ? null : {{/isNullable}}Array.from(value.{{name}} as Set<any>){{/uniqueItems}}{{^uniqueItems}}value.{{name}}{{/uniqueItems}},
{{/isArray}}
{{^isDateType}}{{^isDateTimeType}}{{^isArray}}
'{{baseName}}': value.{{name}},
{{/isArray}}{{/isDateTimeType}}{{/isDateType}}
{{/isPrimitiveType}}
```

But this always generated excess linebreaks:
```typescript
export function TagToJSON(value?: Tag | null): any {
    if (value === undefined) {
        return undefined;
    }
    if (value === null) {
        return null;
    }
    return {
        
        
        'id': value.id,
        
        
        'name': value.name,
        
    };
}
```

Reply:
Okay, I've managed to split the template into multiple lines. Turns out that "if each line contains only a **single** mustache expression, no additional linebreak is inserted" :)